solutions/selection_sort.py

```diff
@@ -0,0 +1,82 @@
+import random
+import time
+
+# O(1) space complexity, O(n^2) time complexity
+def selection_sort_in_place(list):
+
+    for i in range(len(list)):
+
+        min_element = list[i]
+
+        for j in range(i, len(list)):
+
+            if list[j] < min_element:
+                min_element = list[j]
+
+        temp_element_0 = list[i]
+
+        min_element_ind = list.index(min_element)
```

Comment:
Change `min_element_ind` to `min_element_index` for the following reasons:
1. Consistent with the longer "element" usage. Note that "min" and "max" abbreviations are fine since they are extremely common.
2. The `ind` abbreviation of index is not very common. And auto-completion should make it easy for students to use more descriptive variable names.